### PR TITLE
Fixed #25388 -- make it possible to skip migrations during tests

### PR DESCRIPTION
**Pull request by viciu**
https://code.djangoproject.com/ticket/25388

### CODE REVIEW

File: django/test/runner.py [hunk 1]
```diff
@@ -711,6 +711,8 @@ def setup_databases(verbosity, interactive, keepdb=False, debug_sql=False, paral
         first_alias = None
         for alias in aliases:
             connection = connections[alias]
+            # make it possible to skip migrations during tests
+            connection.settings_dict['MIGRATE'] = connection.settings_dict['TEST'].get('MIGRATE', True)
```
Comment (berkerpeksag) [inline django/test/runner.py]: This should probably be done in the application code by setting `DATABASE['TEST']['MIGRATE']` as described in the ticket.

File: django/db/migrations/loader.py [hunk 2]
```diff
@@ -64,6 +64,10 @@ def load_disk(self):
         self.unmigrated_apps = set()
         self.migrated_apps = set()
         for app_config in apps.get_app_configs():
+            # make it possible to skip migrations during tests
+            if self.connection and not self.connection.settings_dict.get('MIGRATE', True):
```
Comment (berkerpeksag) [inline django/db/migrations/loader.py]: Adding a

```py
# pseudo code
if settings.SETTING_NAME_HERE is False:
    return None
```

check to `migrations_module()` would make the code much testable. Or you've already tried and it didn't work for you?

**Comment by berkerpeksag** [comment]
Could you also add a test to `tests/migrations/test_loader.py` and update documentation (see `docs/ref/settings.txt`) for the new setting?

**Comment by timgraham** [comment]
Closing due to inactivity. Please send a new PR if you can update it per the comments.

**Pull request closed without merging**